### ISSUE TITLE
Code coloring can get confused by the 'less than' operator

### ISSUE THREAD

**Issue by gjsjohnmurray** [doublefint/vscode-cos#39]
Try this code:
```
YJMbad1 ;
 s a=1
 i a<1 q
 s a=1
tag ; comment
 w $zversion
 i a'="" w (a'="")
 q
```
Unless you add at least one space between the < and the 1 on the third line the code colorer seems to treat <1 as the start of an HTML / XML tag, so it colors the remaining code incorrectly.

Add the space to see things as they should be.

**Comment by daimor** [comment]
Yes, unfortunately currently code coloring too bad. We need the better parser for the code.

**Comment by doublefint** [comment]
Hi, John! I agree with @daimor, there is only initial support for COS ( CDL ) and very primitive support for M language
